alws/database.py

```diff
@@ -15,7 +15,11 @@
 
 DATABASE_URL = settings.database_url
 
-engine = create_async_engine(DATABASE_URL, poolclass=NullPool)
+engine = create_async_engine(
+    DATABASE_URL, poolclass=AsyncAdaptedQueuePool,
+    pool_size=10, max_overflow=5, pool_pre_ping=True, pool_recycle=1200,
```

Comment:
Out of curiosity, what are the reasons for not using the default values for `pool_size` and `max_overflow`?

Reply:
Previously we've seen that they are not enough. So I prefer a bit bumped numbers